innertube/src/main/java/com/zionhuang/innertube/InnerTube.kt

```diff
@@ -139,7 +140,13 @@ class InnerTube {
                     } else it
                 },
                 videoId = videoId,
-                playlistId = playlistId
+                playlistId = playlistId,
+                playbackContext =
+                    if (client.useSignatureTimestamp) {
+                        PlayerBody.PlaybackContext(PlayerBody.PlaybackContext.ContentPlaybackContext(
+                            signatureTimestamp = YoutubeJavaScriptPlayerManager.getSignatureTimestamp("")
```

Comment:
The doc states:
> Params:
videoId – the video ID used to get the JavaScript base player file (an empty one can be passed, even it is not recommend in order to spoof better official YouTube clients)

Is there a reason not to pass the videoId here (since we have it 4 lines before)